frontend/src/app/header/header.component.html

```diff
@@ -1,9 +1,11 @@
 <header class="head nav-bar">
     <ly-toolbar position="relative" bg="primary" [elevation]="5" class="auto-height">
-      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo" routerLink="/home">
+      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo vspacing" routerLink="/home">
```

Comment:
I need a little help with this. I think I'm stupid.

I want a little bit of spacing between the top/bottom of the Ford logo and the toolbar, but padding and margins don't seem to do anything??

Reply:
@sloan-seamus you're adding a class called `vspacing` but that's not defined anywhere...

use the existing margin classes - if you want more space above / below on the toolbar add these: `mtm mbm`
margin-top-medium, margin-bottom-medium, these are defined in styles.scss